src/Campaigns/resources/admin/components/CampaignDetailsPage/campaignSchema.ts

```diff
@@ -0,0 +1,44 @@
+import {__} from '@wordpress/i18n';
+import {JSONSchemaType} from 'ajv';
+import {CampaignInputFields} from './types';
+
+const campaignSchema: JSONSchemaType<CampaignInputFields> = {
```

Comment:
I'd recommend pulling this from the REST API. That way you are actually doing the validation on the server as well, and it can be updated by things like extensions. You can grab it by making an `OPTIONS` request to the endpoint.

Reply:
Cool idea, thanks!